.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+  "plugins": ["prettier-plugin-tailwindcss"],
+  "semi": true,
+  "singleQuote": false,
+  "tabWidth": 2
+}
```

Comment:
You may not need prettier -- some of the eslint plugins handle that already.

Reply:
I've kept prettier to use the prettier-tailwind plugin, which handel's automatic class sorting for tailwind.